app/models/user.js

```diff
@@ -1,16 +1,15 @@
-const { UUIDV4 } = require('sequelize');
-
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define(
-    'users',
+    'User',
     {
-      id: { type: DataTypes.UUID, defaultValue: UUIDV4, allowNull: false, primaryKey: true },
       firstName: { type: DataTypes.STRING, allowNull: true },
       lastName: { type: DataTypes.STRING, allowNull: true },
       email: { type: DataTypes.STRING, allowNull: false, unique: true },
       password: { type: DataTypes.STRING, allowNull: false }
     },
     {
+      tableName: 'users',
+      freezeTableName: 'users',
```

Comment:
I think that this option is Boolean, please check the [documentation](https://sequelize.org/master/manual/model-basics.html) please. In fact, I think we can avoid it

Reply:
Really sorry, fixed it! :)